### DISCUSSION TITLE
Is it expected not to be able to import pyrfc.pyrfc?

### DISCUSSION THREAD

**Discussion by bluckau** [SAP/PyRFC#312] [General]
We have code that imports the following:

from pyrfc.pyrfc import Connection, Decimal

I can no longer import pyrfc.pyrfc after 2.7.0.   I can still import pyrfc and it gives me access to Connection. But Decimal is not there as far as i can tell.  I might not need it but I didn't want to change my code unless I can verify this is a change that was intentional (is Decimal no longer supported?)

Thanks,

**Comment by bsrdjan** [comment]
Internal binary component of PyRFC is renamed to _cyrfc and this should work:

```python
from pyrfc._cyrfc import Connection
```

Imports from internal component are not recommended because the internal name could be changed in the future. Recommended usage is

```python
from pyrfc import Connection, Decimal
```

Decimal is supported and  export was disabled because of the bug. `import Decimal` will work again in next release, coming soon.
The workaround in the meantime:

```python
from decimal import Decimal
```

see b9e76cb

**Comment by bsrdjan** [comment]
Fixed in 2.8.0
